tooling/lsp/src/requests/completion.rs

```diff
@@ -1900,13 +1914,13 @@ fn get_array_element_type(typ: Type) -> Option<Type> {
     }
 }
 
-fn get_type_struct_id(typ: &Type) -> Option<TypeId> {
+fn get_type_type_id(typ: &Type) -> Option<TypeId> {
```

Comment:
Should probably use `.follow_bindings_shallow` here to handle type variables as well. 

You will need to add the Alias case to that function though.

Reply:
Done!

Oh, I was about to say that `Alias` is already handled by this function, though I guess it must be handled by `follow_bindings_shallow`, right? I'm not sure I'd like to make that change in this PR because that's also used in the compiler and maybe it breaks something. I'll create a follow-up PR to try it, though!